eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -43,10 +43,10 @@ class ContentTypeHandler extends AbstractInMemoryPersistenceHandler implements C
     protected function init(): void
     {
         $this->getGroupTags = static function (Type\Group $group) { return ['type-group-' . $group->id]; };
-        $this->getGroupKeys = static function (Type\Group $group) {
+        $this->getGroupKeys = function (Type\Group $group) {
```

Comment:
Why removing `static`?

Reply:
to use the escapeForCacheKey function we need to change from a static function to normal